components/server/src/auth/generic-auth-provider.ts

```diff
@@ -809,7 +812,35 @@ export class GenericAuthProvider implements AuthProvider {
     };
 }
 
-export type VerifyResult = TosFlow.WithIdentity | TosFlow.WithUser;
+interface VerifyResult {
```

Comment:
The shape of `TosFlow` was used as result, thus moving it here, while getting rid of everything from session's storage. This is now basically the return type of the verify function in the auth provider.